Bricks/cerealize/cerealize.h

```diff
@@ -212,7 +204,7 @@ class CerealBinaryFileParser {
     if (fi_.peek() != std::char_traits<char>::eof()) {  // This is safe with any next byte in file.
       std::unique_ptr<T_ENTRY> entry;
       si_(entry);
-      processor(*entry.get());
+      processor(std::move(*entry.get()));
```

Comment:
Ok, my bad, it's not `const`.
But this still looks like hack for me, I understand `std::move`'ing `unique_ptr`'s, but not their internal pointers.